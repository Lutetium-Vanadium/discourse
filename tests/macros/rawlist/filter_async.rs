fn main() {
    let q = inquisition::questions![rawlist { async filter: todo!() }];
}
