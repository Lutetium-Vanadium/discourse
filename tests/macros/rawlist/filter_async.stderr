error: option `filter_async` does not exist for kind `rawlist`
 --> $DIR/filter_async.rs:2:53
  |
2 |     let q = inquisition::questions![rawlist { async filter: todo!() }];
  |                                                     ^^^^^^
